src/AccessibilityInsights.SharedUxTests/Settings/ConfigurationModelTests.cs

```diff
@@ -19,7 +19,9 @@ namespace AccessibilityInsights.SharedUxTests.Settings
     [TestClass()]
     public class ConfigurationModelTests
     {
-        public static string folderPath = Path.Combine(DirectoryManagement.sUserDataFolderPath, "ConfigurationTests");
+        public static readonly FixedConfigSettingsProvider testProvider = FixedConfigSettingsProvider.CreateDefaultSettingsProvider();
```

Comment:
We could provide some temp folder instead for our unit tests if that feels more natural

Reply:
IIRC, each test run iteration gets its own working directory. Could we leverage that?

---
In reply to: [298770644](https://github.com/microsoft/accessibility-insights-windows/pull/407#discussion_r298770644) [](ancestors = 298770644)